Bottom: 4150deb715bc68715ae4176d1d01a24d1b3abc24
Top:    bda1ecd0bc3d2f999043364042c02e44bfad43db
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:20 +0000

Reduce tolerances a smidgin


---

diff --git a/src/netclass.c b/src/netclass.c
index 54c6d6e..6003f1b 100644
--- a/src/netclass.c
+++ b/src/netclass.c
@@ -31,16 +31,16 @@ static char *netclass_names[10] =
 
 static Coord clearances[10][10] =
   {
-    {MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3)}
+    {MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3)}
   };
